oceanwaters/doc/setup_dev_env.md

```diff
@@ -214,7 +208,7 @@ wget https://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
 sudo apt-get update
 sudo apt-get upgrade
 ```
```

Comment:
My understanding was that we had this step so users could upgrade their Gazebo version from 9.0 to one that supports the API that ow_dynamic_terrain uses. If the user already has Gazebo 11.x installed, is this step still necessary? @Samahu might know.

Reply:
I don't know what Gazebo version do you get with a fresh installation of ROS noetic on Ubuntu 20.04. If it  is 11+ which is probably the case then we don't need the user to perform this step. Yet, the described step sets up the user machine to correctly receive updates and fixes for Gazebo continuously published by OSRF. So I see no harm in keep these instructions.